fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/LaunchFragment.java

```diff
@@ -30,34 +30,27 @@ public View onCreateView(
       @Nullable ViewGroup container,
       @Nullable Bundle savedInstanceState) {
 
-    mLaunchFragBinding = LaunchFragmentBinding.inflate(inflater, container, false);
+    binding = LaunchFragmentBinding.inflate(inflater, container, false);
 
-    mHomeViewModel = HomeActivity.obtainViewModel(requireActivity());
-
-    mLaunchFragBinding.setViewModel(mHomeViewModel);
-    mLaunchFragBinding.setLifecycleOwner(getActivity());
-
-    return mLaunchFragBinding.getRoot();
-  }
-
-  @Override
-  public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
-    super.onViewCreated(view, savedInstanceState);
+    viewModel = HomeActivity.obtainViewModel(requireActivity());
+    binding.setLifecycleOwner(getActivity());
```

Comment:
I'm being an Android-newbie here:
my understanding is that this stuff normally happens implicitly.
Why do we need to manually inflate and set the activity, in this context ?
Is it because it's the entry point ?

Reply:
I don't think there is a way to do the inflation implicitly 😅 

What we do here, is the use of [View Binding](https://developer.android.com/topic/libraries/view-binding)
It is a nice feature that generate code based on the activity xml so that we can get rid of the `findViewById()` 

As for the `setLifecycleOwner` call, this might not be needed. I'm looking into that.

Reply:
I don't understand the use cases I found. I think we can safely remove it.